src/metabase/models/field.clj

```diff
@@ -167,7 +168,9 @@
                                                (:fk_target_field_id field))]
                                 (:fk_target_field_id field)))
         id->target-field (u/key-by :id (when (seq target-field-ids)
-                                         (filter i/can-read? (db/select Field :id [:in target-field-ids]))))]
+                                         (filter i/can-read? (-> Field
+                                                                 (db/select :id [:in target-field-ids])
+                                                                 (hydrate   :table)))))]
```

Comment:
extra whitespace? Also personal preference but

```clojure
(-> (db/select Field :id [:in target-field-ids])
    (hydrate :table))
```

is the way we do it everywhere else and personally I think that would be a little more readable here :)

Reply:
one other thought: we should probably go ahead and remove the `:table` after we use it for `can-read?` purposes. Otherwise it will get returned as part of the API response for every Field, which could potentially dramatically increase the size of the response

Reply:
😆 I added that space because I figured that's what you'd do. I'll remove it.

👍 on removing table after as well

Reply:
I’ve never seen the two spaces like that before. Is that a convention for threading macros? I see how it could make sense